documentation/API-design-guidelines.md

```diff
@@ -1568,6 +1573,22 @@ The decision on the API-level scopes was made within the [Identity and Consent M
 The scopes will always be those defined in the API Specs YAML files. Thus, a scope would only provide access to all endpoints and resources of an API if it is explicitly defined in the API Spec YAML file and agreed in the corresponding API subproject. 
 
 
+### 11.7 API access restriction (Based on token & client Id)
```

Comment:
I would prefer to have more general section title - if agree, then it should be modified also in ToC

```suggestion
### 11.7 Resource access restriction 
```